docker-app/qfieldcloud/filestorage/tests/test_attachments.py

```diff
@@ -37,6 +37,20 @@ def setUp(self):
             attachments_file_storage="webdav",
         )
 
+    def _check_file_exists_on_storage(self, file: File, file_storage_name: str) -> None:
```

Comment:
```suggestion
    def assertFileOnStorage(self, file: File, file_storage_name: str) -> None:
```

I think we can even do it like this?